Cargo.toml

```diff
@@ -37,26 +34,26 @@ regex-fancy = ["syntect/regex-fancy"] # Use the rust-only "fancy-regex" engine
 
 [dependencies]
 atty = { version = "0.2.14", optional = true }
-ansi_term = "^0.12.1"
-ansi_colours = "^1.0"
+ansi_term = "0.12.1"
+ansi_colours = "1.0.1"
```

Comment:
Why would we change this from `^1.0` to `1.0.1`?

Reply:
Can we please revert these changes and only keep the `tempfile`/`dirs-next` ones?

Reply:
These were a side effect of `cargo update`. Let me manually revert